openreview/conference/invitation.py

```diff
@@ -310,11 +310,12 @@ def __init__(self, conference, reveal_authors, reveal_submission, hide_fields):
             if not reveal_authors:
                 content['authors'] = {'values': ['Anonymous']}
                 content['authorids'] = {'values-regex': '.*'}
-            for field in hide_fields:
-                content[field] = {
-                    'values-regex': '.*',
-                    'required': False
-                }
+            if hide_fields:
```

Comment:
can we default `hide_fields` to be an empty array so we avoid this if?

Reply:
Done.